sphinxcontrib/confluencebuilder/builder.py

```diff
@@ -24,6 +24,7 @@
 import io
 import sys
 
+
```

Comment:
(optional) While I believe you are updating the code to be more PEP 8 compliant, this change is not specifically related to this commit. I would normally suggest if you want to make changes like this is to put them in a separate commit; however, the change is trivial enough that it should not prevent consideration for this pull request.